File: 771_jewels_and_stones/jewels_and_stones.js
// JS: https://leetcode.com/problems/jewels-and-stones/submissions/1177500417/
// TS: https://leetcode.com/problems/jewels-and-stones/submissions/1177498046/
function numOfJewsAndStones(jewels, stones) {
    let count = 0;

    for(let i = 0; i < stones.length; i++) {
        if(jewels.includes(stones[i])) {
            count++;
        }
    }

    return count;
}

File: 771_jewels_and_stones/jewels_and_stones.py
# Py3: https://leetcode.com/problems/jewels-and-stones/submissions/1177502326/
def jewels_and_stones(jewels: str, stones: str) -> int:
    count = 0

    for i in range(len(stones)):
        if stones[i] in jewels:
            count += 1
    
    return count